package.json

```diff
@@ -12,7 +12,7 @@
     "web-vitals": "^2.1.2"
   },
   "scripts": {
-    "start": "PORT=4000 react-scripts start",
+    "start": "react-scripts start",
```

Comment:
Switch to using an `.env` file-based approach for configuring the port as the syntax you were using for setting the environment variable via the npm script doesn't work on Windows.